integration/container/logs_test.go

```diff
@@ -124,6 +127,12 @@ func testLogs(t *testing.T, logDriver string) {
 		},
 	}
 
+	pollTimeout := time.Second * 10
+	if testEnv.OSType == "windows" {
+		// hcs can take longer than 10s to stop a container.
+		pollTimeout = time.Second * 60
+	}
```

Comment:
Do we need the same timeout as was used in https://github.com/moby/moby/pull/44893 ? (starting to wonder if there's some central location where we can define this (but I also don't want to make it "more than needed", so perhaps it's fine - mostly thinking if we can make it easier to fine all of these in case we need to update them again in future)).

Reply:
Hm, I think the root case is the same, so I guess it'd make sense to have the timeouts the same.

Reply:
60s seems to be enough, but having a 15s bigger timeout won't hurt us, as most of the time we expect the test to pass.